### PR TITLE
45750 About NMs Page

### PR DESCRIPTION
**Pull request by Shirin14926**
Code changes, Unit test and UI changes for About tab
Also, added comment about the location of the instructions for NM Integration Tests as suggested by Nev

### CODE REVIEW

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/About.cshtml [hunk 1]
```diff
@@ -1,119 +1,122 @@
 ﻿@{
     Layout = "~/Views/Shared/_MSILayout.cshtml";
-    ViewData["Header"] = "About";
+    ViewData["Header"] = "Notice to Mariners";
```
Comment (usha13529) [inline UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/About.cshtml:3]:
```suggestion
    ViewData["Header"] = "Notices to Mariners";
```

**Review by usha13529 — CHANGES_REQUESTED** [review]
Few review comments to fix please.

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/About.cshtml (hunk 1 above)
Comment (Shirin14926) [reply UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/About.cshtml]: Done as expected

File: Tests/AutoTests/AccessibilityTest/accessebility-noticetomarineabout.spec.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import { test, expect, chromium, Page, Browser, BrowserContext } from '@playwright/test';
+
+import * as app from "../../Configuration/appConfig.json";
+import noticetomarine from '../../pageObject/noticetomarine.page';
+import { injectAxe, getViolations, checkA11y, Options} from 'axe-playwright'
+
+test.describe("MSI Notice to Marine Page Accessibility Test", () => {
```
[the remark this answers is not in the document]
Comment (Shirin14926) [reply Tests/AutoTests/AccessibilityTest/accessebility-noticetomarineabout.spec.ts]: Done as expected

File: Tests/AutoTests/AccessibilityTest/accessebility-noticetomarineabout.spec.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    };
+
+    test.beforeEach(async ({ page }) => {
+        await page.goto(app.url);
+        notice = new noticetomarine(page);
+        await notice.clickToNoticemarine();
+        await notice.clickToNoticemarineAbout();
+    });
+
+    test('Notice to Marine About page should be accessible', async ({ page }) => {
```
[the remark this answers is not in the document]
Comment (Shirin14926) [reply Tests/AutoTests/AccessibilityTest/accessebility-noticetomarineabout.spec.ts]: Have corrected the filename

**Review by usha13529 — APPROVED** [review]
Thanks for the changes @Shirin14926. Looks good to me.

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/About.cshtml [hunk 4]
```diff
@@ -1,119 +1,122 @@
...
 </p>
 <p>
-    <a href="https://www.admiralty.co.uk/AdmiraltyDownloadMedia/NMs/Quick%20guide%20of%20new%20ADMIRALTY%20Notices%20to%20Mariners%20tracings.pdf" target="_blank">View the quick guide to Notices to Mariners tracings</a>
+    <a href="~/nms/about/Quick guide of new ADMIRALTY Notices to Mariners tracings.pdf" target="_blank" aria-label="View the quick guide to Notices to Mariners tracings PDF opens in new tab">View the quick guide to Notices to Mariners tracings</a>
 </p>
-
-<h4>
+<!--<p>
+    <a href="https://www.admiralty.co.uk/publications/publications-and-reference-guides/general-publications-and-reference-guides" target="_blank">How to keep your ADMIRALTY Products Up-to-Date (NP294)</a>' 10th Edition, published 1 June 2017 for further information.
+</p>-->
```
Comment (barrie-cooper) [inline UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/About.cshtml:46]: Please remove commented out link:
```suggestion
```

**Review by barrie-cooper — COMMENTED** [review]
Just a few minor points

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/About.cshtml [hunk 5]
```diff
@@ -1,119 +1,122 @@
...
-    <a href="https://www.youtube.com/watch?v=obRmBWhnZEo&list=PLIyHcA-twBuNFDcas706D6vEsmXCTPKtU" target="_blank">Watch the support films &gt;</a>
+    <a href="https://www.youtube.com/watch?v=obRmBWhnZEo&list=PLIyHcA-twBuNFDcas706D6vEsmXCTPKtU" target="_blank" aria-label="Watch the support films here opens in new tab">Watch the support films here</a>
 </p>
-
-<h4>
-    New Notices to Mariners tracings
-</h4>
+<hr aria-hidden="true" />
+<h2>
+    New Notice to Mariners tracings
```
[the remark this answers is not in the document]
Comment (Shirin14926) [reply UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/About.cshtml]: Done, Thanks

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/About.cshtml [hunk 6]
```diff
@@ -1,119 +1,122 @@
...
+</h2>
 <p>
-    Please see our publication '<a href="https://www.admiralty.co.uk/publications/publications-and-reference-guides/general-publications-and-reference-guides" target="_blank">How to keep your ADMIRALTY Products Up-to-Date (NP294)</a>' 10th Edition, published 1 June 2017 for further information.
+    Please see our publication <a href="https://www.admiralty.co.uk/publications/publications-and-reference-guides/general-publications-and-reference-guides" aria-label="How to keep your ADMIRALTY Products Up-to-Date (NP294) opens in new tab">'How to keep your ADMIRALTY Products Up-to-Date (NP294)'</a> 10th Edition, published 1 June 2017 for further information.
```
[the remark this answers is not in the document]
Comment (Shirin14926) [reply UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/About.cshtml]: Removed, Thanks

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/About.cshtml [hunk 7]
```diff
@@ -1,119 +1,122 @@
...
+</p>
 <p>
     ADMIRALTY charts and publications should be maintained so that they are fully up-to-date for the latest safety-critical navigational information. The ADMIRALTY Notices to Mariners service provides all the data you need to maintain our products.
 </p>
 <p>
     ADMIRALTY NMs contain all the corrections, alterations and amendments for the UKHO's worldwide series of ADMIRALTY charts and publications. They are published weekly in several formats but best known as the weekly booklets, which are despatched directly from the UKHO.
 </p>
 <p>
-    <a href="https://www.admiralty.co.uk/AdmiraltyDownloadMedia/NMs/ADMIRALTY-NMs-Guidance-Notes-Sept2015.pdf" target="_blank">View the guidance notes for Notices to Mariners here</a>
+    <a href="~/nms/about/ADMIRALTY-NMs-Guidance-Notes-Sept2015.pdf" target="_blank" aria-label="View the guidance notes for Notice to Mariners here PDF opens in new tab">View the guidance notes for Notice to Mariners here</a>
```
[the remark this answers is not in the document]
Comment (Shirin14926) [reply UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/About.cshtml]: Done, Thanks

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/About.cshtml [hunk 8]
```diff
@@ -1,119 +1,122 @@
...
+<h2>
     ADMIRALTY Notices to Mariners Online
-</h4>
-<p>
-    The ADMIRALTY Notices to Mariners Online service enables mariners to download NMs directly from this website.
-</p>
+</h2>
 <p>
-    To access NMs Online, select the appropriate tabs on this page.
+    The ADMIRALTY Notices to Mariners Online service enables mariners to download NMs directly from this website. NMs Online can be accessed on this website under ‘Notice for Mariners documents’.
```
[the remark this answers is not in the document]
Comment (Shirin14926) [reply UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/About.cshtml]: Done, Thanks

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/About.cshtml (hunk 4 above)
Comment (Shirin14926) [reply UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/About.cshtml]: Removed,Thanks

**Review by barrie-cooper — APPROVED** [review]
LGTM, thanks @Shirin14926

**Pull request merged**